src/indexPage/index.js

```diff
@@ -8,17 +8,43 @@ import { isEuCountry, showHideEuElements } from '../common/footer-checks';
 import GTM from '../common/gtm';
 import { load as loadLang, showBanner } from '../common/lang';
 import { moveToDeriv } from '../common/utils/utility';
-import { getTokenList } from '../common/utils/storageManager';
+import { get as getStorage, set as setStorage, remove, getTokenList } from '../common/utils/storageManager';
 import { createUrl } from '../common/utils/tools';
 import '../common/binary-ui/dropdown';
+import BotLanding from './react-components/BotLanding.jsx';
 
 const renderElements = () => {
-    ReactDOM.render(<Logo />, document.getElementById('binary-logo'));
-    ReactDOM.render(<Footer />, document.getElementById('footer'));
-    isEuCountry().then(isEu => showHideEuElements(isEu));
-    showBanner();
+    const today = new Date().getTime();
+    const expirationDate = today + 1000 * 60 * 60 * 24 * 21;
 
-    $('#shop-url').attr('href', createUrl({ subdomain: 'shop', path: 'collections/strategies', isNonBotPage: true }));
+    const temp = getStorage('setDueDateForBanner');
+
+    const elements = document.querySelectorAll('#notification-banner,#main,#footer,#header,#topbar');
+    if (!temp) {
+        ReactDOM.render(<BotLanding />, document.getElementById('bot-landing'));
+        setStorage('setDueDateForBanner', expirationDate);
+        for (let i = 0; i < elements.length; i++) {
+            elements[i].style.display = 'none';
+        }
+        document.getElementById('bot-landing').style.display = 'block';
```

Comment:
Isn't it possible to set the style dynamically by adding classnames? and put the styles in a stylesheet file?